src/main/java/org/stellar/sdk/PathPaymentStrictSendOperation.java

```diff
@@ -0,0 +1,225 @@
+package org.stellar.sdk;
+
+import com.google.common.base.Objects;
+import org.stellar.sdk.xdr.Int64;
+import org.stellar.sdk.xdr.OperationType;
+import org.stellar.sdk.xdr.PathPaymentStrictSendOp;
+
+import java.util.Arrays;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Represents <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html#path-payment" target="_blank">PathPaymentStrictSend</a> operation.
```

Comment:
The doc is not updated it but I believe the URL will looks like this:
```suggestion
 * Represents <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html#path-payment-strict-send" target="_blank">PathPaymentStrictSend</a> operation.
```